src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -16,11 +16,11 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManage
 {
     internal class FeatureManagementKeyValueAdapter : IKeyValueAdapter
     {
-        private FeatureFilterTracing _featureFilterTracing;
+        private FeatureFlagTracing _featureFlagTracing;
 
-        public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracing)
+        public FeatureManagementKeyValueAdapter(FeatureFlagTracing featureFilterTracing)
         {
-            _featureFilterTracing = featureFilterTracing ?? throw new ArgumentNullException(nameof(featureFilterTracing));
+            _featureFlagTracing = featureFilterTracing ?? throw new ArgumentNullException(nameof(featureFilterTracing));
         }
 
         public Task<IEnumerable<KeyValuePair<string, string>>> ProcessKeyValue(ConfigurationSetting setting, Uri endpoint, Logger logger, CancellationToken cancellationToken)
```

Comment:
Is this PR stale? Parsing is still being done inline with writing configuration.

Reply:
I thought I would be merging the dotnet schema PR first then resolving conflicts here, so I hadn't updated this yet. I forgot to mention that